src/app/Http/Requests/LikeRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class LikeRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'movie_id' => 'int|required',
```

Comment:
Ubaci svuda i validaciju da proveris da li prosledjeni  movie_id/comment_id postoji u bazi.
https://laravel.com/docs/5.1/validation#rule-in
Istrazi i malo na netu kako ovo ide